scripts/rff/dc_generated/report.json

```diff
@@ -0,0 +1,1671 @@
+            "file": "WeatherVariability_Counties.csv",
+            "lineNumber": "3480965"
+          }]
+        }]
+      }]
+    }, {
+      "counterKey": "StatsCheck_MaxPercentFluctuationGreaterThan100",
+      "problemPoints": [{
+        "date": "1983-08",
+        "values": [{
+          "value": 2.151410146859976,
+          "locations": [{
+            "file": "WeatherVariability_Counties.csv",
+            "lineNumber": "2082365"
+          }]
+        }]
+      }, {
+        "date": "1983-09",
+        "values": [{
+          "value": 6.483798452524039,
+          "locations": [{
+            "file": "WeatherVariability_Counties.csv",
+            "lineNumber": "2085473"
+          }]
+        }]
+      }],
+      "percentDifference": 201.37
+    }]
+  }, {
+    "placeDcid": "geoId/31109",
+    "statVarDcid": "Skewness_DailyMinTemperature",
+    "measurementMethod": "",
+    "observationPeriod": "P1Y",
+    "scalingFactor": "",
+    "unit": "",
+    "validationCounters": [{
+      "counterKey": "StatsCheck_MaxPercentFluctuationGreaterThan500",
+      "problemPoints": [{
+        "date": "2006",
+        "values": [{
+          "value": -0.010254619671748235,
+          "locations": [{
+            "file": "WeatherVariability_Counties.csv",
+            "lineNumber": "205133"
+          }]
+        }]
+      }, {
+        "date": "2007",
+        "values": [{
+          "value": -0.2927127251258263,
+          "locations": [{
+            "file": "WeatherVariability_Counties.csv",
+            "lineNumber": "208241"
+          }]
+        }]
+      }],
+      "percentDifference": -2754.45
+    }]
+  }, {
+    "placeDcid": "geoId/31109",
+    "statVarDcid": "Skewness_DailyMinTemperature",
+    "measurementMethod": "",
+    "observationPeriod": "P1M",
+    "scalingFactor": "",
+    "unit": "",
+    "validationCounters": [{
+      "counterKey": "StatsCheck_3_Sigma",
+      "problemPoints": [{
+        "date": "2000-11",
+        "values": [{
+          "value": 1.4588306133563702,
+          "locations": [{
+            "file": "WeatherVariability_Counties.csv",
+            "lineNumber": "2725721"
+          }]
+        }]
+      }, {
+        "date": "2004-03",
+        "values": [{
+          "value": 1.3008077768179087,
+          "locations": [{
+            "file": "WeatherVariability_Counties.csv",
+            "lineNumber": "2850041"
+          }]
+        }]
+      }]
+    }, {
+      "counterKey": "StatsCheck_MaxPercentFluctuationGreaterThan500",
+      "problemPoints": [{
+        "date": "2000-09",
+        "values": [{
+          "value": 0.0014613527518052322,
+          "locations": [{
+            "file": "WeatherVariability_Counties.csv",
+            "lineNumber": "2719505"
+          }]
+        }]
+      }, {
+        "date": "2000-10",
+        "values": [{
+          "value": -0.9825292147122897,
+          "locations": [{
+            "file": "WeatherVariability_Counties.csv",
+            "lineNumber": "2722613"
```

Comment:
Just to sanity check this, the variation in Skewness_DailyMinTemperature from one month to another is OK?

This is for Lancastar County, Nebraska (https://datacommons.org/browser/geoId/31109)

Reply:
To elaborate on that... from 2000-09 to 2000-10 the value goes from 0.0014613527518052322 to -0.9825292147122897.

Reply:
Hannah confirmed that this is not an issue so we should be good to go!

Reply:
Thanks!